src/screens/Home/Home.tsx

```diff
@@ -39,39 +59,113 @@ function Home(props: HomeProps) {
   const numCustomersToCollect = useSelector(selectCustomersToCollect)?.length || 0;
   const allSites: SiteRecord[] = useSelector(selectAllSitesInformation) || [];
   const currentSite: SiteRecord = useSelector(selectCurrentSiteInformation) || EMPTY_SITE;
+  const user = useSelector(selectCurrentUser);
   const isOnline = useSelector(selectIsOnline) || false;
   const lastUpdated = formatDateStringToLocal(useSelector(selectLastUpdated));
 
+  // This'll eventually be calculated via a selector that sums the profit of Mee Panyar over all periods
+  // Tiffany will have the profit calculation done on her Financial Summary PR, so we can add it after that
+  const amountOwed = isOnline ? 6000 : 0;
+
+  const getAmountOwedCard = () => {
+    const amountOwedCardInfo: CardPropsInfo[] = [{
+      number: amountOwed.toString(),
+      label: 'Owed to MP',
+      unit: 'Ks'
+    }]
+
+    const addButton = <IconButton
+      component={Link}
+      to='/home'
+      size="small"
+      className={classes.buttonPrimary}
+    >
+      <AddIcon />
+    </IconButton>
+
+    return (
+      // Same margin bottom that Typography applies as gutterBottom
+      <div style={{ marginBottom: '.35em' }}>
+        <OutlinedCardList
+          highlighted={amountOwed !== 0}
+          info={amountOwedCardInfo}
+          rightIcon={amountOwed === 0 ? <CheckCircleOutlineIcon /> : addButton}
+        />
+      </div>)
+  }
+
   return (
     <BaseScreen rightIcon="user">
-      <div className={classes.header}>
-        <SiteMenu currentSite={currentSite} sites={allSites} />
-        <div className={classes.network}>
-          {isOnline ? <CloudQueueIcon color="primary" /> : <CloudOffIcon color="secondary" />}
-          <Typography className={classes.network} variant="body1">
-            Last Connected to Network <br />
-            {lastUpdated}
-          </Typography>
+      <BaseScrollView>
+        <div className={classes.header}>
```

Comment:
Maybe check with Ashley to see if she intended for this header to be fixed? Since the headers are sticky on customer and inventory screens, I wonder if this should be the same.